external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaDataConsumer.scala

```diff
@@ -628,6 +628,8 @@ private[kafka010] object KafkaDataConsumer extends Logging {
     if (TaskContext.get != null && TaskContext.get.attemptNumber >= 1) {
       val cacheKey = new CacheKey(topicPartition, kafkaParams)
 
+      logDebug(s"Task re-attempt detected, invalidating consumers with key $cacheKey")
+
```

Comment:
Once we add above log, comment is being redundant. I'd make the code lines also symmetric with producer side - two lines without no newline, logDebug -> invalidateKey.

Reply:
Makes sense, changed.